src/Polly.Core/Telemetry/ResilienceEventSeverity.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Polly.Telemetry;
+
+/// <summary>
+/// The severity of reported resilience event.
+/// </summary>
+public enum ResilienceEventSeverity
```

Comment:
Dunno, in this case why would someone report event with `None` value? They could as well not report it. How should we handle it?

As for `Debug` and `Critical` we can add it but there won't be any usages.

Reply:
Sure we might not use them, but someone with a custom policy built on top of the API might.

I figured an explicit None would just be good as the default value, otherwise Information would be used as the default value, which would probably be a mistake. I think LogLevel is the same (unless it's called Off or something).

Reply:
Ok, better safe than sorry.